spinalcordtoolbox/deepseg/models.py

```diff
@@ -188,6 +188,13 @@
 
 # List of task. The convention for task names is: action_(animal)_region_(contrast)
 # Regions could be: sc, gm, lesion, tumor
+CROP_MESSAGE = (
+    'To crop the data you can first segment the spinal cord using the contrast agnostic model, This could be '
```

Comment:
Tiny punctuation typo:
```suggestion
    'To crop the data you can first segment the spinal cord using the contrast agnostic model. This could be '
```